Neos.Neos/Resources/Private/Fusion/RawContent/NodeProperties.fusion

```diff
@@ -0,0 +1,16 @@
+prototype(Neos.Neos:RawContent.NodeProperties) < prototype(Neos.Fusion:Component) {
+    node = null
+
+    renderer = afx`
+        <Neos.Fusion:Loop items={props.node.nodeType.fullConfiguration.properties} itemKey="propertyName" itemName="propertyConfiguration">
```

Comment:
this getter https://github.com/neos/neos-development-collection/blob/fd83b396c9d5641bf1c983a1cdd449fa38214552/Neos.ContentRepository/Classes/Domain/Model/NodeType.php#L336 

> Should only be used internally.
> Instead, use the hasConfiguration()/getConfiguration() methods to check/retrieve single configuration values.

but `getConfiguration("properties")` wont work in eel.
i know we are the core and allowed to use internal stuff, but if someone looks at it he wont know that and mistake it as api...

what do you think?

Reply:
Nice catch adjusted ... turns out `nodeType.properties` already does all i need. 

See https://github.com/neos/neos-development-collection/blob/fd83b396c9d5641bf1c983a1cdd449fa38214552/Neos.ContentRepository/Classes/Domain/Model/NodeType.php#L424